zulipterminal/config/keys.py

```diff
@@ -4,10 +4,12 @@
     'HELP': {
         'keys': {'?'},
         'help_text': 'Show/hide help menu',
+        'is_random_tip': False,
     },
     'GO_BACK': {
         'keys': {'esc'},
         'help_text': 'Go Back',
+        'is_random_tip': True,
```

Comment:
I have changed the implementation (2nd commit, with flags) solely due to the fact that mypy from TravisCI does not like dictionaries containing both boolean and string values (type checking fails) at the same time. I have just found out that a work around exists -  TypedDict. Therefore I will try to stick with adding just a boolean.

Reply:
If you have the development environment, then you should be able to run mypy locally too. I'm not sure what error you might be getting here, but I'm not sure why a `bool` would be an issue, as `keys` is already a `Set[str]`, not a `str`.

I'm not sure we would benefit from `TypedDict here, since `KEY_BINDINGS` is not highly exposed and we don't explicitly type it.

Reply:
I was getting 'error: Value of type "object" is not indexable'  [here](https://travis-ci.org/zulip/zulip-terminal/builds/494479130?utm_source=github_status&utm_medium=notification). I have to admit, such error caught me a little off guard.

If we do not need other flags for now, then I think using TypedDict is the better solution (albeit I wish it was not necessary since it seems a little superfluous).

Reply:
That error does sound strange; I'm not sure where mypy thinks the object is coming from, without digging into the code further at this point.